cmd/kubeadm/app/apis/kubeadm/v1beta3/conversion.go

```diff
@@ -32,21 +32,10 @@ func Convert_v1beta3_InitConfiguration_To_kubeadm_InitConfiguration(in *InitConf
 		return err
 	}
 	err = Convert_v1beta3_ClusterConfiguration_To_kubeadm_ClusterConfiguration(&ClusterConfiguration{}, &out.ClusterConfiguration, s)
-	// Make roundtrip / fuzzers happy
-	// TODO: Remove with v1beta2 https://github.com/kubernetes/kubeadm/issues/2459
-	out.DNS.Type = ""
 	return err
 }
 
-// Convert_kubeadm_DNS_To_v1beta3_DNS is required since Type does not exist in the DNS struct
-// TODO: Remove with v1beta2 https://github.com/kubernetes/kubeadm/issues/2459
-func Convert_kubeadm_DNS_To_v1beta3_DNS(in *kubeadm.DNS, out *DNS, s conversion.Scope) error {
-	return autoConvert_kubeadm_DNS_To_v1beta3_DNS(in, out, s)
-}
-
 // Convert_v1beta3_ClusterConfiguration_To_kubeadm_ClusterConfiguration is required due to the missing
```

Comment:
this is not a complete and read-able sentence after the removing of the below line.